config/monitoring/prometheus/apps/prometheus-configs.yaml

```
@@ -1815,3 +1840,100 @@ data:
               severity: warning
               instance: feast-operator-controller-manager
+  llama-stack-k8s-operator-recording.rules: |
+    groups:
+      - name: SLOs - Llama Stack K8s Operator
+        rules:
+        - expr: |
+            absent(up{job="llama-stack-k8s-operator"}) * 0 or vector(1)
+          labels:
+            instance: llama-stack-k8s-operator-controller-manager
+          record: probe_success
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)